packages/trip-details/src/types.ts

```diff
@@ -0,0 +1,44 @@
+import moment from "moment";
+// HACK: Probably ok to import the file using a relative path as it is used during dev only.
+// Also prettier does not recognize the import type syntax.
+// eslint-disable-next-line prettier/prettier
+import type { Itinerary } from "../../types/src";
```

Comment:
This should be fixed before merging, as the note implies.  Will either @opentripplanner/types/src  -or- #/types/src work?

Reply:
Thanks for pointing out this one. `@opentripplanner/types` seems to work also so I used that in https://github.com/opentripplanner/otp-ui/pull/281/commits/1fa98e123757bdfce2c79781f885d8261283c21a.

Reply:
After changing the import to use the "cleaner" syntax, it looks like the `types` package needs to be published first for the CI to pass, so I opened #291.